src/test-run/commands/options.js

```diff
@@ -59,6 +59,100 @@ export class OffsetOptions extends ActionOptions {
     }
 }
 
+export class ScrollOptions extends OffsetOptions {
+    constructor (obj, validate) {
+        super();
+
+        this.forceCenter = false;
```

Comment:
I'm not sure that `forceCenter` is a best naming there. I guess it can be something like 'scrollToCenter' or `centered` .. Please discuss it with @VasilyStrelyaev